modules/json-api.php

```diff
@@ -32,6 +34,63 @@
 	Jetpack_Sync::sync_posts( __FILE__, $sync_options );
 }
 
+/**
+ * On each page load, we'll check to see if new updates for the site are available,
+ * and save the updates data to a jetpack option. We skip this check on subsites in
+ * a multi-site network.
+ */
+function jetpack_json_api_get_site_updates() {
+
+	// if we are not on the main site, we don't need to calculate updates
+	// we can save a blank array
+	if ( ! is_main_site() ) {
+		Jetpack_Options::update_option( 'wp_updates', array() );
+		return;
+	}
+
+	$update_data = wp_get_update_data();
+	if ( isset( $update_data['counts'] ) ) {
+		$updates = $update_data['counts'];
```

Comment:
Wouldn't this `$updates` be overwritten on the next line ? where $updates['wp_version'] is set?